docs/configuration/network_filters/mongo_proxy_filter.rst

```diff
@@ -25,6 +26,31 @@ access_log
   path is specified no access logs will be written. Note that access log is also gated by
   :ref:`runtime <config_network_filters_mongo_proxy_runtime>`.
 
+fault
+  *(optional, object)* If specified, the filter will inject faults based on the values in the object.
+
+Fault configuration
+-------------------  
+
+Configuration for MongoDB fixed duration delays. Once delay is injected all of the multiplexed
```

Comment:
What is unclear about this text is that I assume the Mongo Filter will work very similar to the HTTP Fault Filter; however, after talking about it in person that isn't the case. 

Please elaborate on the first set of data that arrives with no active timer enabled will get the full delay duration and all subsequent data arriving on that connection up until the timer is 'complete' will be apart of that delay.

Reply:
Also, do we have an issue tracking making this fault filter 'smarter' and to rework it so either a consistent delay is applied every new call of OnData or per operation?

Reply:
@RomanDzhabarov just in case you didn't see this comment.

Reply:
yup, https://github.com/lyft/envoy/issues/530. I'll link to the ticket and explain what's missing.